src/components/Patient/ManagePatients.tsx

```diff
@@ -948,7 +891,18 @@ export const PatientManager = () => {
                         });
                         return data ?? null;
                       },
-                      parse: preventDuplicatePatientsDuetoPolicyId,
+                      parse: (data) => {
```

Comment:
Let's switch to using useQuery here.

Reply:
@Jacobjeevan There is no api request in parsing why to use useQuery

Reply:
@Jacobjeevan Need to change to all useTanStackQueryInstead to useQuery in both manage patients and dischargedPatientslist . Im holding this PR for long time. Let change them in another issue .

Reply:
> @Jacobjeevan Need to change to all useTanStackQueryInstead to useQuery in both manage patients and dischargedPatientslist . Im holding this PR for long time. Let change them in another issue .

We are doing slow migration  - switching over whenever a file is modified, rather than opening an issue that switches all at once.

Reply:
@Jacobjeevan But it is throwing an erro saying tht useQuery cannot  be used an inside a function. Can you confirm where do want me to use useQuery

Reply:
https://react.dev/reference/rules/rules-of-hooks

Reply:
Check the link Rithvik sent; but let us know if you are still having issues with using useQuery.

Reply:
@Jacobjeevan  Don’t call Hooks inside loops, conditions, nested functions, or try/catch/finally blocks. Couldnt able to use usequery

Reply:
you shouldn't be calling hooks inside functions (other than react components) in the first place. you can achieve it this way:

```
const { refetch: exportAsCSV } = useQuery({
  queryKey: [...],
  qyeryFn: query(...),
  enabled: false,
});


// in export action:
action: async () => {
 const data = await exportAsCSV();
 return data;
}
```